pybamm/experiments/experiment.py

```diff
@@ -134,9 +143,18 @@ def read_string(self, cond):
             electric = self.convert_electric(cond_list[:idx])
             time = None
             events = self.convert_electric(cond_list[idx + 1 :])
+        elif "Run" in cond:
+            # e.g. Run US06
+            cond_list = cond.split()
+            electric = (drive_cycles[cond_list[1]][:, 1], "Drive")
+            # Set time and period to 1 second
+            # because drive cycles are sampled at 1Hz frequency
```

Comment:
This is not always true. For drive cycles, the time and period should be set to "None" and then the difference between times in the drive cycle itself should be used.